### PR TITLE
fix: resource description in configuration column overlaps

### LINKED ISSUE #1125: Resources description in Configuration column of session list record overlaps when resource exceeds certain values.
**Describe the bug**
문제가 발생하는 상황에 대한 요약 설명을 적어주세요.
When there are lots of resources allocated in one session, then some texts in the record of configuration column which represents the session overlaps.

**To Reproduce**
문제를 재현하기 위한 순서를 자세히 적어주세요:
1. Go to 'Sessions' page
2. Click on 'FINISHED' tab.
3. Select GPU description in configuration column from any record by using developer tools in each Web browser.
4. Input arbitrary numbers (e.g. 1000.00, 556.00) in the selected text.
4. See the error.

**Expected behavior**
원래 기대했던 동작은 어떤 것인지 명확하게 적어주세요.
When text width increases, column width should be changed, not overlapping.

**Screenshots**
Just take a screenshot in your clipboard or copy a picture file and paste it here (Ctrl+V/Cmd+V) so that GitHub automatically attach and insert it.
가능하다면, 문제 상황을 보여주는 스크린샷을 첨부해주세요.
클립보드에 스크린샷을 찍거나 이미지 파일을 복사해서 여기에 붙여넣으면(Ctrl+V/Cmd+V) GitHub이 자동으로 이곳에 첨부 및 삽입해줍니다.
overlapping-in-configuration-column

**Client:**
 - OS: [e.g. macOS 10.14, Windows 10 (1903)]
 - App version: [e.g. 19.08.3] (if you are using a GUI client app)
 - Browser version: [e.g. Chrome 76, Safari, Edge] (if you are using a GUI web console)
 - Python version: [e.g. 3.7.4] (if you are using the Python SDK)
 - NodeJS version: [e.g. 12.0] (if you are using the Javascript SDK)

**Server:**
 - OS: [e.g. Ubuntu 18.04, CentOS 7.6, macOS 10.14]
   - Output of `uname -a`: [e.g. `Linux node01 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`]
 - Backend.AI version: [e.g. 19.09.0]
 - Python version: [e.g. 3.6.8]
 - Installation method: [e.g. on-premise, using cloud.backend.ai service, used scripts/install-dev.sh]

**Additional context**
추가적으로 문제 상황을 설명할 수 있는 다른 맥락을 기술해주세요.

### PR DESCRIPTION
**Pull request by yujung7768903**
Resolve #1125 

### 1. Change the arrangement of items in Configuration column
It would be better to arrange resource descriptions(CPU, RAM, GPU) to be grouped together.    

* Before (height of grid-cell: 93.250px)
Screen Shot 2021-12-05 at 3 58 18 PM

* After (height of grid-cell: 108px)
Screen Shot 2021-12-05 at 4 00 53 PM

### 2. Notify that there is no folder mounted
Screen Shot 2021-12-05 at 4 17 23 PM

### 3. Show all mounted folder when multiple folders are mounted
* Before
Screen Shot 2021-12-05 at 4 26 59 PM
* After
Screen Shot 2021-12-05 at 4 29 27 PM

### 4. Add bottom border to last row of session list
* Before
Screen Shot 2021-12-06 at 11 54 36 PM

* After
Screen Shot 2021-12-06 at 11 53 38 PM

### CODE REVIEW

File: src/components/backend-ai-session-list.ts [hunk 1]
```diff
@@ -377,6 +383,16 @@ export default class BackendAiSessionList extends BackendAIPage {
     }
   }
 
+  // Styling the shadow DOM
```
Comment (lizable) [inline src/components/backend-ai-session-list.ts]: Any reasons for appending this styles?

File: src/components/backend-ai-session-list.ts [hunk 2]
```diff
@@ -1301,28 +1317,48 @@ export default class BackendAiSessionList extends BackendAIPage {
...
+        <div class="layout horizontal center flex">
+          <div class="layout horizontal center configuration">
+            ${rowData.item.mounts.length > 0 ? html`
+              <wl-icon class="fg green indicator">folder_open</wl-icon>
+              <button class="mount-button"
+                @mouseenter="${(e) => this._createMountedFolderDropdown(e, rowData.item.mounts)}"
+                @mouseleave="${() => this._removeMountedFolderDropdown()}"
+              >
+                ${rowData.item.mounts.map((item, index, list) => {
+                  if (index + 1 === list.length) {
```
Comment (lizable) [inline src/components/backend-ai-session-list.ts]: if you want to add a single space when the condition doesn't match, then how about using tenary operation?

**Review by lizable — CHANGES_REQUESTED** [review]
I left comments below

File: src/components/backend-ai-session-list.ts (hunk 1 above)
Comment (yujung7768903) [reply src/components/backend-ai-session-list.ts]: I want to add bottom border to last <tr> of table in shadow root.
Screen Shot 2021-12-06 at 11 53 38 PM
This element is in shadow-root of vaadin-grid, So I added style in this way.
vaadin-grid-style

File: src/components/backend-ai-session-list.ts (hunk 2 above)
Comment (yujung7768903) [reply src/components/backend-ai-session-list.ts]: Okay. I'll use it!

File: src/components/backend-ai-session-list.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-session-list.ts:386]: I don't think this code insertion is necessary. when you add this style, then you need to add every component that uses `vaadin-grid`, which would bring degradation of performance in rendering a long tabular view.
I guess that it's a `vaadin-grid` issue, not originated from us, so I suggest you to remove this code.

**Review by lizable — APPROVED** [review]
@yujung7768903
I guess we can simplify string modification by replacing a single quote to a double quote, and then use the [`parse()` function in JSON object.](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse).
(Please check out the code [here](https://github.com/lablup/backend.ai-webui/pull/1170/files#diff-4d843242713270c38dd580c57115bfd4160a27e2b9922086d2bbb83efe5d1895R1317-R1318).)
And I recommend you to update your PR branch sync to the main branch frequently. :)

LGTM.

**Comment by inureyes** [comment]
LGTM

**Review by inureyes — APPROVED** [review]
LGTM

**Comment by inureyes** [comment]
I'll merge the PR after 3 days of approval from majority of reviewers.

**Pull request merged**